helpers.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs');
+const path = require('path');
+
+/**
+ * Recursively get directories in the path. Adjust maximum depth by setting maxDepth (default: -1)
+ * Paths will be relative or absoluate based on startPath
+ * @param {string} startPath the path to start from
+ * @param {number} maxDepth maximum depth to recurse
+ * @returns {array} array of paths as strings
+ */
+module.exports.getDirs = (startPath, maxDepth = -1) => {
```

Comment:
Should move to a separate package or module and be tested.
Checked using `console.time`; takes ~4ms on `playground` directory.